ui/src/components/deployments/ReplicaSelect.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import Select from "react-select";
+
+class ReplicaSelect extends Component {
+  render() {
+    const { currentReplica, deployment, updateCurrentReplicaFunc } = this.props;
+
+    if (
+      deployment === undefined ||
+      deployment.spec === undefined ||
+      deployment.spec.replicas < 1
+    ) {
+      return <></>;
```

Comment:
Shouldn't we display the button and grey it out? Maybe with `No Available Replicas`?

Reply:
Thanks, good point! We now render a button with `disabled={true}` if `spec.replicas` equal 0.

<img width="1624" alt="Screenshot 2023-03-31 at 11 23 42" src="https://user-images.githubusercontent.com/128683/229081870-ff4beff9-768f-4e28-8334-8839146a2298.png">